rust/azure_iot_operations_services/src/azure_device_registry/models/asset.rs

```diff
@@ -894,7 +894,7 @@ impl From<DiscoveredEventGroup> for base_client_gen::DiscoveredAssetEventGroup {
     fn from(value: DiscoveredEventGroup) -> Self {
         base_client_gen::DiscoveredAssetEventGroup {
             events: value.events.option_vec_into(),
-            default_events_destinations: value.default_events_destinations.option_vec_into(),
+            default_destinations: value.default_events_destinations.option_vec_into(),
```

Comment:
I think there will also need to be an update in managed_azure_device_registry in the connectors crate to update to the new naming